web/libs/editor/src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx

```diff
@@ -424,16 +424,17 @@ const RootTitle: FC<any> = observer(
               </Elem>
             )}
           </Elem>
-          <RegionControls
-            hovered={hovered}
-            item={item}
-            entity={props.entity}
-            regions={props.children}
-            type={props.type}
-            collapsed={collapsed}
-            hasControls={hasControls && isArea}
-            toggleCollapsed={toggleCollapsed}
-          />
+          {item.hideable && (
+            <RegionControls
```

Comment:
I'm not sure we really want to remove all controls under this condition. There's functionality like `"Copy Region Link"`, and I don't yet understand whether it requires removal or will operate as intended if retained.